src/network/connection_handler_task.cpp

```diff
@@ -0,0 +1,69 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// connection_handler_task.cpp
+//
+// Identification: src/network/connection_handler_task.cpp
+//
+// Copyright (c) 2015-17, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#include "network/connection_handler_task.h"
+#include "network/connection_handle.h"
+#include "network/connection_handle_factory.h"
+
+namespace peloton {
+namespace network {
+
+ConnectionHandlerTask::ConnectionHandlerTask(const int task_id)
+    : NotifiableTask(task_id), new_conn_queue_(QUEUE_SIZE) {
+  int fds[2];
+  if (pipe(fds)) {
+    LOG_ERROR("Can't create notify pipe to accept connections");
+    exit(1);
+  }
+  new_conn_send_fd_ = fds[1];
+  RegisterEvent(fds[0], EV_READ|EV_PERSIST,
+                METHOD_AS_CALLBACK(ConnectionHandlerTask, HandleDispatch),
+                this);
+}
+
+void ConnectionHandlerTask::Notify(int conn_fd) {
+  new_conn_queue_.Enqueue(conn_fd);
+  char buf[1];
+  buf[0] = 'c';
```

Comment:
What is `c` here? Can we define that as a constant somewhere?

Reply:
Legacy.